app/templates/views/why-notify.html

```diff
@@ -0,0 +1,62 @@
+{% extends "content_template.html" %}
+{% from "components/table.html" import mapping_table, row, text_field, edit_field, field with context %}
+{% from "components/sub-navigation.html" import sub_navigation %}
+
+{% block per_page_title %}
+  {{ _('Why use GC Notify') }}
+{% endblock %}
+
+{% block content_column_content %}
+
+<div class="pl-24">
+  <h1 class="heading-large">{{ _('Why use GC Notify') }}</h1>
+</div>
+<div class="pl-24">
+  <p>{{ _('GC Notify is designed in a way that meets the needs of government services.') }}</p>
+</div>
+<div class="pl-24">
+  <img src="/static/images/tick.png" class="float-left" style="width: 4rem; height: 4rem; margin-left:-5rem;"/>
+  <h2 class="heading-medium">{{ _('Build modern, cloud-first services') }}</h2>
+</div>
+<div class="pl-24">
+  <p>{{ _('GC Notify is a scalable cloud platform with a robust API that allows you to connect to other services. For example, connect to electronic mailing list software like Listserv or other applications to send password resets, email confirmations, or responses to forms.') }}</p>
+</div>
+<div class="pl-24">
+  <img src="/static/images/tick.png" class="float-left" style="width: 4rem; height: 4rem; margin-left:-5rem;"/>
+  <h2 class="heading-medium">{{ _('Comply with communications requirements') }}</h2>
+</div>
+<div class="pl-24">
+  <p>{{ _('GC Notify provides a bilingual interface and bilingual options for Government of Canada signature branding in email messages by default to ensure you are compliant with the Federal Identity Program.') }}</p>
+</div>
+<div class="pl-24">
+  <img src="/static/images/tick.png" class="float-left" style="width: 4rem; height: 4rem; margin-left:-5rem;"/>
+  <h2 class="heading-medium">{{ _('Protect personal information') }}</h2>
+</div>
+<div class="pl-24">
+  <p>{{ _('GC Notify does not store contact lists and only stores information about recipients for 7 days. You can also hide sensitive information by redacting personal details from messages after they’ve been sent, so that only recipients see their personal information.') }}</p>
+  <p><a href="{{ url_for('main.privacy') }}">{{ _('Learn more about privacy protection') }}</a></p>
+</div>
+<div class="pl-24">
+  <img src="/static/images/tick.png" class="float-left" style="width: 4rem; height: 4rem; margin-left:-5rem;"/>
+  <h2 class="heading-medium">{{ _('Keep data secure') }}</h2>
+</div>
+<div class="pl-24">
+  <p>{{ _('GC Notify keeps your account secure with two-factor authentication. This means you’ll need to enter a code that gets sent to your phone or email when you sign in. GC Notify is authorized for Protected A information. Data is encrypted whenever it passes through GC Notify and when it is stored in the system.') }}</p>
+  <p><a href="{{ url_for('main.security') }}">{{ _('Learn more about security') }}</a>.
```

Comment:
```suggestion
  <p><a href="{{ url_for('main.security') }}">{{ _('Learn more about security') }}</a>
```
No final dot in Figma and to be consistent with other links